File: Data Structures & Algorithms/duplicate-integer/submission-1.java
class Solution {
    public boolean hasDuplicate(int[] nums) {
 for(int i = 0; i < nums.length - 1; i++)
 {
    for(int j = i; j < nums.length - 1; j++){

    
    if( nums[i] == nums[j+1]) {
        return true;
    } else {}
    }
 }
 return false;
    }

}

File: Data Structures & Algorithms/top-k-elements-in-list/submission-1.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        dic = {}

        for i in nums:
            if i in dic:
                dic[i] += 1
            else:
                dic[i] = 1
        result = []
        for i in range(k):
            most = max(dic,key = dic.get)
            result.append(most)
            dic.pop(most)
        return result